README.md

```diff
@@ -42,18 +42,18 @@ of fictional attendees][personas] we created to help design the workshop.
 
 # Learning objectives
 
-At the end of the workshop I will ...
+At the end of the workshop you will ...
 
-* ... be able to create a new project that contains an analysis script
-  which takes raw data and produces figures and tables from it without
+* … be able to create a new project that contains an analysis script
```

Comment:
what is this unicode magic? Seriously, how badly will this break if someone is brave enough to look at this in Notepad(or whatever it is people use) on Windows?